### PR TITLE
docs: Add documentation for CPU parameter

### LINKED ISSUE #3317: Write documentation for the `cpus` parameter
The same way we documented options related to `memory` in `docs/memory.md`, let's document `cpus`.

### PR DESCRIPTION
**Pull request by sboeuf**
This new document describes the available options for the parameter `--cpus`, how to use them and what is their usage.

Fixes #3317

### CODE REVIEW

File: docs/cpu.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+# CPU
+
+Cloud Hypervisor has many options when it comes to the creation of virtual
+CPUs. This document aims to explain what Cloud-Hypervisor is capable of and
```
Comment (rbradford) [inline docs/cpu.md:4]: You change hyphenation in the middle of the paragraph. The project is called "Cloud Hypervisor" compare and contrast `git grep -c Cloud Hypervisor` with `git grep -c Cloud-Hypervisor`. If you want to refer to the binary do so as `cloud-hypervisor`

**Review by rbradford — APPROVED** [review]

**Comment by rbradford** [comment]
Make sure to update the website to link to this when it is merged.

File: docs/cpu.md (hunk 1 above)
Comment (sboeuf) [reply docs/cpu.md:4]: Definitely!

**Comment by sboeuf** [comment]
>Make sure to update the website to link to this when it is merged.

Will do!

**Pull request merged**